local registry upgrade clarification (#646)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/configure-local-registry.adoc

Review the following limitations on support for local registries: 

* Runtime Fabric supports local registries on Self-Managed Kubernetes only.
* You must use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or later.
* You can use a local registry only if you create a new Runtime Fabric instance of version 1.12.0 or later.
+

> If you set up a local registry, you can upgrade to a version later than 1.12.0 without needing to re-install Runtime Fabric.

Reviewer: ```suggestion
If you set up a local registry, you can upgrade to a version later than 1.12.0 without reinstalling Runtime Fabric.
```

+

> You cannot add a local registry to an existing Runtime Fabric instance (regardless of its version) even if you upgrade that installation to 1.12.0 or later.

Reviewer: ```suggestion
You cannot add a local registry to any version of an existing Runtime Fabric instance, even if you upgrade that installation to 1.12.0 or later.
```
